### DISCUSSION TITLE
Tracking event detections/alerts over MQTT

### DISCUSSION THREAD

**Discussion by bobemoe** [blakeblackshear/frigate#14196] [General Support] [answered]
I have set up my Detections/Alerts so that Frigate is properly classifying events of interest to me as **Alerts** and leaving the rest as **Detections**, ideal.

I processing the MQTT frigate/events to try and generate only notifications of Alerts however am having trouble finding the info in the MQTT stream.

I feel I'm missing something? I expected to see a `"type":"detection"` or `"type":"alert"` in there somewhere but no?

Am I looking in the wrong place or misunderstanding? 

I think maybe it is not classified until the event has ended? Although I don't see why I couldn't change from detection to alert once a qualifying object enters a qualifying zone.

What am I missing?

Cheers :)

**Comment by NickM-27** [comment, marked as answer]
you should be using the `frigate/reviews` topic instead of the events topic

**Reply by bobemoe** [reply]
Doh! So it is, and its nicely documented too. Sorry for missing it. I guess `frigate/reviews` is a new topic since 0.14?

**Reply by NickM-27** [reply]
yes
